media/transcoder/transcoder_test.go

```diff
@@ -180,6 +199,33 @@ func writeTestGCSFile(t *testing.T, dstBucket string, srcBucket string, srcObjec
 	}
 }
 
+func checkGCSFileExists(t *testing.T, bucketName string, fileName string) {
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		t.Fatalf("storage.NewClient: %v", err)
+	}
+	defer client.Close()
+
+	ctx, cancel := context.WithTimeout(ctx, time.Second*10)
+	defer cancel()
+
+	it := client.Bucket(bucketName).Objects(ctx, nil)
```

Comment:
You can use [`Bucket.Object`](https://pkg.go.dev/cloud.google.com/go/storage#BucketHandle.Object) to get the object directly, rather than iterating through every object in the bucket.

You can then use [`ObjectHandle.Attrs`](https://pkg.go.dev/cloud.google.com/go/storage#ObjectHandle.Attrs) to check if the object exists.